src/share/poudriere/common.sh

```diff
@@ -7791,7 +7792,7 @@ prepare_ports() {
 	if [ $SKIPSANITY -eq 0 ]; then
 		msg "Sanity checking the repository"
 
-		for n in repo.txz digests.txz packagesite.txz; do
+		for n in repo.{pkg,txz} digests.{pkg,txz} packagesite.{pkg,txz}; do
```

Comment:
I believe `{pkg,txz}` does not work in `/bin/sh`.

Reply:
Indeed.